patches/api/0392-Also-load-resources-from-LibraryLoader.patch

```diff
@@ -0,0 +1,52 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nick Hensel <nickhensel25@icloud.com>
+Date: Sun, 28 Aug 2022 23:44:18 +0200
+Subject: [PATCH] Also load resources from LibraryLoader
+
+
+diff --git a/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java b/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
+index 345394132df70593800127d34a38f8f8a4dafe00..d09811ba3aedadc20f8b37883fb5a02d81177284 100644
+--- a/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
++++ b/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
+@@ -10,6 +10,7 @@ import java.net.URL;
+ import java.net.URLClassLoader;
+ import java.security.CodeSigner;
+ import java.security.CodeSource;
++import java.util.ArrayList;
+ import java.util.Collection;
+ import java.util.Collections;
+ import java.util.Enumeration;
+@@ -90,12 +91,31 @@ public final class PluginClassLoader extends URLClassLoader { // Spigot
+ 
+     @Override
+     public URL getResource(String name) {
+-        return findResource(name);
++        // Paper start
++        var resource = findResource(name);
```

Comment:
No `var` please

Reply:
The var is still here, there was two var in fact